index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <h1>Welcome To Microverse!</h1>
+  <p>
+    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Similique,
+    laudantium!
+  </p>
+
+  <h1>What is Pair Programming</h1>
```

Comment:
Good job so far! 👍🏻
There are some issues that you still need to work on it.
Please make sure to use only one h1 element on the page because it's better for website SEO. Also, it's a good coding practice.